src/content/docs/merge-queue/setup.mdx

```diff
@@ -7,65 +7,63 @@ import { Image } from "astro:assets"
-
-The Merge Queue works by ensuring each PR meets all defined conditions before
-it's merged. It checks for PR status, branch conditions, approval reviews, and
-other set criteria, ensuring that your main branch is always stable and free of
-bugs that might occur due to unreviewed code. Furthermore, by managing PRs
-queue, Mergify eliminates the need for manual tracking and merging, freeing up
-your team to focus on what's essential — building quality software.
-
-In this guide, we'll walk you through the process of setting up your first
-Merge Queue, explain how to add PRs to it, and show you how to configure the
-conditions that the PRs must meet before being merged. Whether you're new to
-Mergify or looking to make your process more efficient, this guide will give
-you the knowledge you need to leverage the power of Merge Queue.
-
-## Understanding the Default Merge Queue
-
-Mergify provides a default merge queue named `default` for every repository as
-soon as you set up Mergify. This queue works out-of-the-box and uses your
-repository's [branch protection
+Mergify’s Merge Queue automates and secures your PR merging process. It ensures
+each PR meets all conditions (status checks, branch rules, reviews, etc.)
+before merging, so your main branch stays stable — without manual tracking or
+coordination.
+
+:::note
+For more details about merge queues and their functions, you can refer to our
+[merge queue introduction](./).
+:::
+
+## Default Merge Queue
+
+Every repo connected to Mergify gets a queue automatically. By default, it
+respects your GitHub [branch protection
 settings](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
-defined in GitHub.
+and
+[rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets).
 
-For instance, if you have branch protection rules that require passing CI
-checks before merging, the `default` merge queue respects these rules. It means
-Mergify will only merge pull requests that pass these checks and are in the
-`default` queue.
+## Adding a Pull Request to the Merge Queue
 
-Understanding this default queue is essential as it forms the basis of how
-Mergify handles your pull requests, ensuring a streamlined and orderly merge
-process.
+To add a pull request to the merge queue, use [the `queue`
+command](/commands/queue). This adds your PR to the merge queue as long as it
+matches the queue conditions.
 
-For more details about merge queues and their functions, you can refer to our
-[merge queue introduction](./).
+<Image src={queueCommandScreenshot} alt="Mergify queue command" />
+
+Remember, the PR will only be merged once the conditions defined in the
+`merge_conditions` field are met.
+
+If you prefer, you can ask Mergify to automatically queue pull requests that
+are ready using workflow automation.
```

Comment:
Is that a reference to `autoqueue`? If yes, is it part of the workflow automation or the merge queue feature? Maybe we can add a link to the settings